Makefile

```diff
@@ -117,14 +117,14 @@ letsencrypt:
 
 tre-stop:
 	echo -e "\n\e[34m»»» 🧩 \e[96mStopping TRE\e[0m..." \
-	&& . ./devops/scripts/check_dependencies.sh \
+	&& . ./devops/scripts/check_dependencies.sh azfirewall \
```

Comment:
Initial thinking was that azure-firewall extension is a base dependency with any TRE development/deployment, just like az cli.  Since, if deploying it's essential you can stop/start it to manage costs.  Having explicit reference here is good idea though.

Reply:
I was running another make command - not start/stop locally and it got me hence thought worth adding.